packages/common/src/store/ui/modals/album-track-remove-confirmation-modal.ts

```diff
@@ -0,0 +1,21 @@
+import { createModal } from './createModal'
+
+export type AlbumTrackRemoveConfirmationModalState = {
+  confirmCallback: (() => void) | undefined
```

Comment:
hm i think we don't want to have fns in redux bc they're not serializable. i ran into this before lemme see if i can find an example of what the move is.

Reply:
https://github.com/AudiusProject/audius-protocol/pull/6608/files

Reply:
Yeah I'm aware that it breaks convention, but I don't necessarily have an idea to fix; do you have something in mind? 
We also do this in like 4 other spots as well 😅
We're gonna be limited by the redux integrated modals and also the CollectionPageProvider also not being able to use hooks lol

Reply:
ah yeah that's annoying... guess i thought that we'd remove the `removeTrackFromPlaylist` prop and just call whatever that is manually from inside the child components?

Reply:
yeah can we just call these in the child components? https://github.com/AudiusProject/audius-protocol/blob/345d3dfaa38cee13021ac3d0da3c3f8172f2828d/packages/web/src/pages/collection-page/CollectionPageProvider.tsx#L858

this might be something to bring up with the team tho. i'm not a fan of the provider pattern and i think we wanna move away from it, but maybe good to confirm.